### PR TITLE
Add AFwizard to conda-forge

### PR DESCRIPTION
**Pull request by dokempf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dokempf** [comment]
@conda-forge/help-python This is ready for review.

File: recipes/afwizard/conda_build_config.yaml
Comment (BastianZim) [inline recipes/afwizard/conda_build_config.yaml:1]: You'll also need to target the channel for your beta build: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

File: recipes/afwizard/meta.yaml
Comment (BastianZim) [inline recipes/afwizard/meta.yaml:63]:
```suggestion
  license_file: LICENSE.md
  dev_url: https://github.com/ssciwr/afwizard
  doc_url: https://afwizard.readthedocs.io/
```

File: recipes/afwizard/meta.yaml
Comment (BastianZim) [inline recipes/afwizard/meta.yaml:28]: This is fixed upstream to 3.4.0. Which one is correct, since you're also upstream?

File: recipes/afwizard/meta.yaml
Comment (BastianZim) [inline recipes/afwizard/meta.yaml:63]: This should also be distributed, since it contains a license file: https://github.com/ssciwr/afwizard/blob/main/COPYING.md

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/afwizard/conda_build_config.yaml
Comment (dokempf) [reply recipes/afwizard/conda_build_config.yaml:1]: Specified the `dev` channel target.

File: recipes/afwizard/meta.yaml
Comment (dokempf) [reply recipes/afwizard/meta.yaml:65]: Added. I did not know about the existence of these additional URLs, very helpful.

File: recipes/afwizard/meta.yaml
Comment (dokempf) [reply recipes/afwizard/meta.yaml:29]: I have recently fixed the underlying problem that introduced the requirement for the pin. The version used in the recipe does not require it anymore.

File: recipes/afwizard/meta.yaml
Comment (dokempf) [reply recipes/afwizard/meta.yaml:65]: I have merged the information from `COPYING.md` into `LICENSE.md` in the last beta release used in this recipe.

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
Looks great. Please ping me once the CI passes and I'll merge it.

**Comment by dokempf** [comment]
@BastianZim CI passed. Thanks a lot for the fast and helpful review.

**Pull request merged**
